portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,21 @@ function addRandomGreeting() {
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+  
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomline() {
+    const lines=['Im a peacock you gotta let me fly','just do it',
+                 'crewmen','clap your hands if you believe', 
+                'FIRE EVERYTHING!','You will be obliterated']
+  
+    // Pick a random line.
```

Comment:
Delete these comments. You typically don't want comments that say *What* the code is doing, you want the what/how to be fairly obvious by reading the code itself. Comments can be useful for saying *Why* a bit of code exists if it's not obvious, e.g. "Return early here to avoid sending an unnecessary HTTP request". If you think the code is hard to read without a comment, consider instead writing a new helper function, e.g. `const movieline = pickRandomLine(lines);`.

Reply:
on it. i took them out and its not hard to read I guess I just like commenting when i can